lib/node_modules/@stdlib/blas/ext/base/scusumpw/lib/ndarray.js

```diff
@@ -21,7 +21,7 @@
 // MODULES //
 
 var float64ToFloat32 = require( '@stdlib/number/float64/base/to-float32' );
-var floor = require( '@stdlib/math/base/special/floor' );
+var floorf = require( '@stdlib/math/base/special/floorf' );
```

Comment:
```suggestion
var floor = require( '@stdlib/math/base/special/floor' );
```

This change is not desired. We want the floor of `N/2` to be in double-precision, as the max safe single-precision floating-point number is very small.